packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1618,10 +1618,10 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final bool cursorOpacityAnimates;
 
-  ///{@macro flutter.rendering.RenderEditable.cursorOffset}
+  /// {@macro flutter.rendering.RenderEditable.cursorOffset}
```

Comment:
Thanks for catching these!

<br>

I have a slight preference toward tiny fixes like this one being in their own PR:
- In the future, someone might be going through the git history and could be confused as to why a merge commit labeled `Fix TabBar crash with SliverAppBar` was making changes in editable_text.dart.
- If this PR is reverted for some reason, it would also revert the documentation fixes.

But the effort put into opening & reviewing a separate PR should be factored in too, so leaving this as-is sounds good to me :)

Reply:
That makes sense. I'll open a new PR the next time. Thanks for your kind suggestion.